### ISSUE TITLE
WCAG 2.0 AA assessment

### ISSUE THREAD

**Issue by nkkl** [18F/cms-hitech-apd#1259]
Our product needs to be accessible! Testing as we go will make our lives a lot easier than waiting until *everything* is done and then testing it in bulk.

### This task is done when...
 - [ ] Key workflows/use cases have been listed
 - [ ] A WCAG 2.0 AA assessment has been completed for each of those workflows
 
Workflows:
- [ ] workflow 1
- [ ] workflow 2

**Comment by nkkl** [comment]
Depends on #1260

**Comment by jeromeleecms** [comment]
Closing this in favor of the team tagging items for accessibility testing after an item/section is complete.
